admin_guide/scheduling/pod_affinity.adoc

```diff
@@ -280,7 +282,7 @@ The following example demonstrates pod anti-affinity for pods with matching labe
 +
 [source,yaml]
 ----
-cat pod-s1.yaml
+$ cat pod-s1.yaml
```

Comment:
For consistency in this article, perhaps `[source,terminal]` should be used here too?